Dockerfile.compose

```diff
@@ -2,37 +2,20 @@
 
 FROM debian:stretch-slim
 
-RUN apt-get -y update
+# Install Dependencies
+RUN apt-get update && apt-get install -y \
+    python3 \
+    python3-pip \
+ && rm -rf /var/lib/apt/lists/*
 
-# Install Python
-RUN apt-get -y install python3
-RUN apt-get -y install apt-transport-https
-RUN apt-get -y install curl
-RUN apt-get -y install gnupg2
-RUN apt-get -y install git
+# Install python dependencies. Do this before copying the entire project to keep cache.
+COPY requirements.txt /tmp/
```

Comment:
This will ensure that image cache is busted if `requirements.txt` ever changes. I also removed the manual clone of srtm.py. It's in PyPi so I have no clue why he was cloning it directly.